nixos/modules/services/torrent/deluge.nix

```
@@ -41,6 +41,7 @@ in {
 
         openFilesLimit = mkOption {
           default = openFilesLimit;
+          type = types.int;
```

According to https://www.freedesktop.org/software/systemd/man/systemd.exec.html#LimitCPU=, `infinity` and `4G:16G` are valid for `LimitNOFILE` (which this option is used for), so:

```suggestion
          type = types.either types.int types.str;
```